src/sparsezoo/model/model.py

```diff
@@ -281,19 +282,50 @@ def validate(
                 )
                 return False
 
-        if self.model_card.path is None:
+        if self.model_card._path is None:
             raise ValueError(
-                "It seems like the `ModelDirectory` was created using "
-                "the `from_zoo_api()` method. Before running `validate()` "
+                "Model card missing! Before running `validate()` "
                 "method, the respective files must be present locally. "
-                "The solution is to call `download()` first."
+                "The solution may be to call `download()` first."
             )
 
         return self.integration_validator.validate(minimal_validation)
 
     def analyze(self):
         raise NotImplementedError()
```

Comment:
This may be outside scope of this PR, but what's this function for? Could it just be deleted?